src/components/logo/Logo.jsx

```diff
@@ -0,0 +1,8 @@
+import logo from "./3.png";
+
+const Logo = (props) => {
```

Comment:
As for me, it is better to use whenever it is possible a named export, not default.